app/src/main/res/values/strings.xml

```diff
@@ -99,4 +99,6 @@
     <string name="notes_empty_error">Notes cannot be empty</string>
     <string name="available_to_be_mentor_mentee">Available to be a:</string>
     <string name="settings">Settings</string>
+    <string name="error_name_too_short">Name should have a minimum of 2 characters</string>
```

Comment:
Is it too much to ask for this string resource to be a formatted string, that accepts an integer in the place of the size of the name (same for the line below)

This is because, if we need to change these constraints, we'll only need to change in the integer resources.

What do you think?

Reply:
I think that's a good idea, it's just gonna be one less thing to update in the future in case we change character count requirements. I'll have to look into how to use formatted strings in XML though. I'll push an update soon.